legacy-bios/mbr.s

```diff
@@ -0,0 +1,31 @@
+        .code16
+        .section .text
+        .globl start
+
+start:
+        jmp $0, $.entry
+.entry:
+        cli
+        xorw %ax, %ax
+        movw %ax, %ss
+        movw %ax, %ds
+        movw $0x7c00, %sp
+        sti
```

Comment:
Last comment, and then it's a +1 from me, I missed this on initial review, but add a 'cld' instruction before printing the string. The 'cld' instruction clears the direction flag which is a good thing to do initially as we don't know the state of the direction flag when loaded. This affects how  `lodsb` loads string data.

```suggestion
        sti
        cld
```